app/collections/collections-manager.spec.js

```diff
@@ -47,4 +64,9 @@ describe('<CollectionsManager />', function() {
     addButton.simulate('click');
     sinon.assert.calledOnce(addToCollectionsSpy);
   });
+
+  it('can add a subject to an empty collection', function() {
+    const addButton = wrapper.find('.search-button');
+    addButton.simulate('click');
```

Comment:
missing assert

Reply:
I think this is missing `sinon.assert.calledTwice(addToCollectionsSpy);` (called twice because the previous test already called the spy once.)

Reply:
Also, this test is just repeating the previous test without any changes.